### PR TITLE
Fe feat/home page conditional rendering

### LINKED ISSUE #4: User Story #4 - Nav bar buttons navigation
[] - As a user, I should be able to navigate to my FieldGuide from the HomePage and the DetailsPage by clicking the "My Field Guide" button.

[] - As a user, I should be able to navigate to my HomePage from the DetailsPage and my FieldGuide by clicking the "Home" button.

### LINKED ISSUE #7: Test - User Story #1 (Image upload)

### LINKED ISSUE #15: Component - HomePage

### LINKED ISSUE #22: Implement Router

### LINKED ISSUE #38: Conditional HomePage rendering
- Change the conditional rendering of the form so that once the image is uploaded, the form disappears & only the image & the "Upload New Image" buttons are shown on the left side of the screen. 

- Create function to RenderMushroomCards for infinite scroll container.

### LINKED ISSUE #39: Add probability function
- Create probability function in order to clean-up the probability for better UI (ie from .97644 to:  97%)

### PR DESCRIPTION
**Pull request by andreasorensen**
# Description

- added formatProbability fn to utils.js file.
- moved userImage state to App component so that user's image stays on homescreen after navigating to other routes
- styled homepage-left-container
- added mockData to use for MushroomCard rendering
- created a renderMushroomCard function to map over each potential match
- moved renderMushroomCard fn to App component so that it can be passed to both the HomePage and the FieldGuide

** note, styling still needs to be done for the mushroom cards, but since we will be passing down the helper fn to the field guide and homepage, josh might have it all done already, so once we see that, we will be ready to go... The homepage-left-container will also need styling.

Closes #15 
Closes #38 
Closes #39 
Closes #7 
Closes #4 
Closes #22 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [ ] Refactoring

# How Has This Been Tested?

Not yet, but will be run through CircleCI

# UI/UX Changes: 

Homepage!

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/MushroomCard/MushroomCard.js [hunk 1]
```diff
@@ -14,8 +15,7 @@ const MushroomCard = ({ image, latinName, commonNames, probability }) => {
         <h2>{latinName}</h2>
         <p>{commonNames.join(", ")}</p>
       </div>
-      <img alt={latinName && commonNames} />
-      {/* need to add src to line 14 */}
+      <img src={image} alt={latinName && commonNames} />
```
Comment (jmartin777) [inline src/Components/MushroomCard/MushroomCard.js:18]: Good refactor on this!

File: src/Components/App/App.js [hunk 2]
```diff
@@ -4,10 +4,27 @@ import './App.css';
...
+      <Link to={`details/${mushroom.id}`} key={mushroom.id}> 
+        <MushroomCard
+            image={mushroom.image}
+            latinName={mushroom.latinName}
+            commonNames={mushroom.commonNames}
+            probability={mushroom.probability}
+        />
+      </Link>
+    ));
+  };
```
Comment (Elise-Jones) [inline src/Components/App/App.js:27]: Great idea moving renderMushroom to App

**Review by Elise-Jones — APPROVED** [review]

File: src/Components/HomePage/HomePage.js [hunk 3]
```diff
@@ -2,39 +2,27 @@ import React, { useState } from "react";
...
-        <div className="homepage-left-container">
-          <img className="user-image" src={userImage} alt="mushroom" />
-          <button
-            className="upload-new-button"
-            onClick={() => onImageUpload(null)}
-          >
-            Upload New Photo
-          </button>
-        </div>
+        <>
```
Comment (jmartin777) [inline src/Components/HomePage/HomePage.js:12]: the left container holds the user image and upload button, and the right container displays mushroom cards?

**Review by jmartin777 — APPROVED** [review]
Looking good and hopefully we can meet in the middle of this code and handle merge conflicts local machine peacefully. Don't think we will have to many

**Pull request merged**